lib/grammar.js

```diff
@@ -207,20 +211,18 @@ var NCNameChar = chars_without(NameChar, ':');
 var NCName = reg(NCNameStartChar, NCNameChar, '*');
 
 /**
-https://www.w3.org/TR/xml-names/#ns-qualnames
-
-```
-[7] QName ::= PrefixedName | UnprefixedName
-				  === (NCName ':' NCName) | NCName
-				  === NCName (':' NCName)?
-[8] PrefixedName ::= Prefix ':' LocalPart
-								 === NCName ':' NCName
-[9] UnprefixedName ::= LocalPart
-									 === NCName
-[10] Prefix ::= NCName
-[11] LocalPart ::= NCName
-```
-*/
+ * https://www.w3.org/TR/xml-names/#ns-qualnames
+ *
+ *     [7] QName ::= PrefixedName | UnprefixedName
+ *     				  === (NCName ':' NCName) | NCName
+ *     				  === NCName (':' NCName)?
+ *     [8] PrefixedName ::= Prefix ':' LocalPart
+ *     								 === NCName ':' NCName
+ *     [9] UnprefixedName ::= LocalPart
+ *     									 === NCName
+ *     [10] Prefix ::= NCName
+ *     [11] LocalPart ::= NCName
+ */
```

Comment:
I really prefer  to stick to the "code fences" (wrapping those things by triple back ticks) way of doing that, since I really dislike this mixture of spaces and tabs in the same line.

I think there is an option for this.

Reply:
Byproduct of other plugin.

Reply:
Just in case, this isn't the plugin; The plugin ignores all comments that don't have a JSDoc tag (unless you enable [`jsdocExperimentalFormatCommentsWithoutTags`](https://github.com/homer0/packages/tree/main/packages/public/prettier-plugin-jsdoc#parse-comments-without-tags))